PowerEditor/src/ScintillaComponent/columnEditor.cpp

```diff
@@ -299,7 +348,6 @@ void ColumnEditorDlg::switchTo(bool toText)
 	::EnableWindow(::GetDlgItem(_hSelf, IDC_COL_INCREASENUM_EDIT), !toText);
 	::EnableWindow(::GetDlgItem(_hSelf, IDC_COL_REPEATNUM_STATIC), !toText);
 	::EnableWindow(::GetDlgItem(_hSelf, IDC_COL_REPEATNUM_EDIT), !toText);
-	::EnableWindow(::GetDlgItem(_hSelf, IDC_COL_FORMAT_GRP_STATIC), !toText);

```

Comment:
Groupbox has no state so EnableWindow should not be used. When used, it can caused some issue with text color.

Reply:
Even in light mode?

Reply:
In light mode it does nothing, but in dark mode it will reset back to black color.

Reply:
It's OK for this PR, because the enabled zone and the disabled zone are quite distinguishable. On the others PR we may make it be able to be disabled under only the light mode.

Reply:
@donho
I think only "Custom Backup Directory" in preference in backup also has this issue.